### PR TITLE
Updated-Googleworkspace

### PR DESCRIPTION
**Pull request by JainM6**
## Purpose of this pull request

This pull request (PR) ...

Issue number: 

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [X] Update Content - Revisions and updating sections
- [ ] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md [hunk 1]
```diff
@@ -27,33 +42,66 @@ If the Source has any issues during any one of these states, it is placed in an
...
+To set up a Google Workspace account, you need to configure Google service account credentials.
 
-You can click on the status icon to open a Health Events panel with
-details on each detected issue.
+### Creating Service Account
 
-## Set up Service Account Credentials
+To create service account credentials, follow the steps below:
+1. Navigate to the [Google Console service account](https://console.cloud.google.com/projectselector2/iam-admin/serviceaccounts?pli=1&supportedpurview=project) page.
+1. Login with your credentials.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md:58]:
```suggestion
1. Log in with your credentials.
```

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md [hunk 2]
```diff
@@ -66,34 +114,26 @@ To configure a Google Workspace Source:
...
-  * **Customer ID**:** The unique ID for the customer's Google Workspace account. In the case of a multi-domain account, to fetch all groups for a customer, fill this field instead of domain. You can also use the `my_customer` alias to represent your account's `customerId`. The `customerId` is also returned as part of the [Users resource](https://developers.google.com/admin-sdk/directory/v1/reference/users).
+  * **Customer ID**. The unique ID for the customer's Google Workspace account. In the case of a multi-domain account, to fetch all groups for a customer, fill this field instead of domain. You can also use the `my_customer` alias to represent your account's `customerId`. The `customerId` is also returned as part of the [Users resource](https://developers.google.com/admin-sdk/directory/v1/reference/users).
   * **Domain**: The domain name. Use this field to get fields from only one domain. To return all domains for a customer account, use the `customer` query parameter instead.
-
-1. When you are finished configuring the Source, click **Submit**.
+12. When you are finished configuring the Source, click **Submit**.
```
Comment (kimsauce) [inline docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md:136]: I think the numbered steps may need to be fixed in this section.

**Review by kimsauce — DISMISSED** [review]
Minor suggestions

File: docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md (hunk 2 above)
Comment (JainM6) [reply docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/google-workspace-source.md:136]: Oh, I tried to fix that, but it looks like the change was not reflected.

**Comment by JainM6** [comment]
> Minor suggestions

Fixed. Thank You.

**Pull request merged**